docs/dev-tools/zombienet/overview.md

```diff
@@ -23,7 +23,7 @@ In order to install Zombienet, there are multiple options available, depending o
 
 === "Using the Executable" 
```

Comment:
Got the ok to keep the tabs for this Installation section. It does seem like there are a lot of `note` and `warning` sections which may result in people skipping over them, etc. My suggestion is to review what the [Google documentation guide](https://developers.google.com/style/notices) says about notices and rework as many of these as you can into regular text. Keep only the really important things as notes and warnings.